### DISCUSSION TITLE
Is there a way to use an older version of Refine (3.18)?

### DISCUSSION THREAD

**Discussion by Robj1925** [refinedev/refine#4481] [Q&A] [answered]
Just wondering if theres a way to use an older version?

**Comment by BatuhanW** [comment, marked as answer]
Hello @Robj1925, you can use following command to generate refine app with v3 version:

```
npm create refine-app@latest -- --branch v3
```

**Reply by MarkoLumbar** [reply]
Hello, im trying to use this version but now i have problems with mui, to be more exact refine-mui probably mui updated dependencies dont work with the old version of refine. Could you help me? I get this errors: export 'default' (reexported as 'generateUtilityClass') was not found in '@mui/base/generateUtilityClass' (possible exports: generateUtilityClass, isGlobalState)
ERROR in ./node_modules/@pankod/refine-mui/dist/esm/index.js 3448:0-4185
export 'default' (reexported as 'generateUtilityClass') was not found in '@mui/base/generateUtilityClass' (possible exports: generateUtilityClass, isGlobalState)

ERROR in ./node_modules/@pankod/refine-mui/dist/esm/index.js 3448:0-4185
export 'default' (reexported as 'generateUtilityClasses') was not found in '@mui/base/generateUtilityClasses' (possible exports: generateUtilityClasses)

ERROR in ./node_modules/@pankod/refine-mui/dist/esm/index.js 3448:0-4185
export 'default' (reexported as 'unstable_composeClasses') was not found in '@mui/base/composeClasses' (possible exports: unstable_composeClasses)
